.github/workflows/scanner-build.yaml

```diff
@@ -94,6 +101,21 @@ jobs:
       if: matrix.name == 'prerelease'
       run: echo "BUILD_TAG=$(make -C scanner --quiet --no-print-directory tag)-prerelease" >> "$GITHUB_ENV"
 
+    - name: Setup Go build environment for -race
+      if: |
+        matrix.goarch == 'amd64'
+          &&
+        (
+          contains(github.event.pull_request.labels.*.name, 'ci-race-tests')
+            ||
+          matrix.name == 'race-condition-debug'
+        )
+      run: echo "RACE=1" >> "$GITHUB_ENV"
```

Comment:
nit: The equiv in main stackrox build uses `RACE=true`, if want to keep things consistent

https://github.com/stackrox/stackrox/blob/d985f327d6fa58622238598124786fcb3bf298cf/.github/workflows/build.yaml#L187

https://github.com/stackrox/stackrox/blob/53dba5c9e4737f7cd64fa86341eaa301a630828c/scripts/go-tool.sh#L14

https://github.com/stackrox/stackrox/blob/53dba5c9e4737f7cd64fa86341eaa301a630828c/scripts/go-tool.sh#L67-L71

Reply:
yeah I thought about that, but I had some trouble getting that to work in the Makefile (definitely not a strong suit for me). I'll try a bit more, and if I can get it, I'll add it here prior to merge

Reply:
got it. turns out it doesn't like `""` in that `ifeq` syntax